migration.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/python
```

Comment:
For portability I'd suggest changing this to:
```py
#!/usr/bin/env python3
```
With this you can drop the `.py` on this script and run `chmod +x migration` to make it a standalone executable.

Reply:
Making the change now

Reply:
I also updated the pr and renamed the script to `aerie_db_migration` to make it a little more readable

Reply:
Sweet! And I guess it's not a big deal anyways since this script is a prototype, yeah? Either way, looks good 👌